client/signup/config/flows.js

```diff
@@ -181,16 +181,16 @@ function removeUserStepFromFlow( flow ) {
 	} );
 }
 
-function getCurrentFlowName( currentUrl ) {
-	// Headstart test - Only consider users from the homepage
-	if ( '/start/en?ref=homepage' === currentUrl && 'headstart' === abtest( 'headstart' ) ) {
+function filterFlowName( flowName ) {
+	const headstartFlows = [ 'blog', 'website' ];
```

Comment:
Let's move this array to be inline in the next line so we don't need a (potentially confusing) variable name.